### PR TITLE
Fix: Open notifications in new tab

### PR DESCRIPTION
**Pull request by irliao**
This will always open notification page in a new foreground tab.

### CODE REVIEW

**Comment by irliao** [comment]
@sindresorhus i believe my pull request will resolve #4

File: Notifier for GitHub.safariextension/main.js [hunk 1]
```diff
@@ -30,11 +30,9 @@
 			var win = safari.application.activeBrowserWindow;
 			var url = 'https://github.com/notifications';
 
-			if (win.activeTab.url) {
-				win.openTab().url = url;
-			} else {
-				win.activeTab.url = url;
-			}
+            var newTab = safari.application.activeBrowserWindow.openTab();
```
Comment (sindresorhus) [inline Notifier for GitHub.safariextension/main.js]: Could just be `var newTab = win.openTab();`

Comment (sindresorhus) [inline Notifier for GitHub.safariextension/main.js]: The indentation also looks off.

File: Notifier for GitHub.safariextension/main.js [hunk 2]
```diff
@@ -30,11 +30,9 @@
 			var win = safari.application.activeBrowserWindow;
 			var url = 'https://github.com/notifications';
 
-			if (win.activeTab.url) {
```
Comment (sindresorhus) [inline Notifier for GitHub.safariextension/main.js]: This logic should already work. It checks if the tab has an URL, and if it doesn't, opens the notifications page in the current tab.

Comment (irliao) [inline Notifier for GitHub.safariextension/main.js]: It seems logically correct; however, when I tested it the notifications page will sometimes override the current page even when the current url does not match notifications page url.

Comment (sindresorhus) [inline Notifier for GitHub.safariextension/main.js]: Yeah, might be a race issue. I've encountered many of those with the Safari extensions API...

**Pull request closed without merging**